### PR TITLE
Accept the new everest-models hook for linting forward models

### PR DESCRIPTION
**Pull request by sregales-TNO**
**Issue**
Running into pydantic errors when trying to validate forward model job file arguments (specifically well swapping)  which uses the new standard 

**Approach**
No error when validating forward models

### CODE REVIEW

**Review by verveerpj — APPROVED** [review]
LGTM!

File: everest/plugins/hook_impl.py [hunk 1]
```diff
@@ -55,6 +55,16 @@ def get_forward_models():
     return None
 
 
+@hookimpl
+def lint_forward_models():
```
[the remark this answers is not in the document]
Comment (sregales-TNO) [reply everest/plugins/hook_impl.py:59]: well swapping is using the new standard we introduced, but I forgot to hook it up to the plugin. So standalone fm_well_swapping was working fine. But when used through Everest config validation it fails not knowing what to do. The previous PR on everest-models side exposed the hook and this is to accept the new plugin hook

[the remark this answers is not in the document]
Comment (verveerpj) [reply everest/plugins/hook_impl.py:59]: If I searchfor `lint_forward_models` it only gives an hit in this file, so it does not seem to be used? Or should it be `lint_forward_model` i.e. without `s`?

[the remark this answers is not in the document]
Comment (sregales-TNO) [reply everest/plugins/hook_impl.py:59]: `lint_forward_model` will fix it

**Pull request merged**